docker-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
This file seems pretty redundant as mentioned above

Reply:
That's a common good practice to hace an entry point script rather than running the binary.